src/ContactManager/Contacts.cs

```diff
@@ -0,0 +1,47 @@
+﻿// <copyright file="Contacts.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace ContactManager
+{
+    public class Contacts
+    {
+        public string name;
+        public string email;
+        public string phone;
+        public string additionalNotes;
+        public Contacts(string name, string email, string phone, string additionalNotes)
```

Comment:
You should also have a empty constructor if you need to validate and add some settings.

Reply:
Changed as required